crates/supervisor/rpc/src/response.rs

```diff
@@ -0,0 +1,140 @@
+//! Supervisor RPC response types.
+
+use alloy_eips::BlockNumHash;
+use alloy_primitives::{ChainId, map::HashMap};
+use kona_protocol::BlockInfo;
+use kona_supervisor_types::SuperHead;
+
+/// Describes superchain sync status.
+///
+/// Specs: <https://github.com/ethereum-optimism/specs/blob/main/specs/interop/supervisor.md#supervisorsyncstatus>.
+#[derive(Debug, Default, Clone, PartialEq, Eq)]
+#[cfg_attr(
+    feature = "serde",
+    derive(serde::Serialize, serde::Deserialize),
+    serde(rename_all = "camelCase")
+)]
+pub struct SupervisorSyncStatus {
+    /// [`BlockInfo`] of highest L1 block.
+    pub min_synced_l1: BlockInfo,
+    /// Timestamp of highest safe block.
+    pub safe_timestamp: u64,
+    /// Timestamp of highest finalized block.
+    pub finalized_timestamp: u64,
+    /// Map of all tracked chains and their individual [`SupervisorChainSyncStatus`].
+    pub chains: HashMap<ChainId, SupervisorChainSyncStatus>,
+}
+
+/// Describes the sync status for a specific chain.
+///
+/// Specs: <https://github.com/ethereum-optimism/specs/blob/main/specs/interop/supervisor.md#supervisorchainsyncstatus>
+#[derive(Debug, Default, Clone, Copy, PartialEq, Eq)]
+#[cfg_attr(
+    feature = "serde",
+    derive(serde::Serialize, serde::Deserialize),
+    serde(rename_all = "camelCase")
+)]
+pub struct SupervisorChainSyncStatus {
+    /// Highest [`Unsafe`] head of chain.
+    ///
+    /// [`Unsafe`]: op_alloy_consensus::interop::SafetyLevel::Unsafe
+    pub r#unsafe: BlockInfo,
```

Comment:
```suggestion
    pub local_unsafe: BlockInfo,
```

Reply:
If you look at the current state in the [golang implementation](https://github.com/ethereum-optimism/optimism/blob/develop/op-service/eth/supervisor_status.go#L17), it still says localUnsafe

Reply:
true, `SafetyLevel` def is also updated in go impl. probably specs need updating. https://github.com/ethereum-optimism/optimism/blob/e6848b15f5dafb3159e993f7aa24844679a44e5b/op-supervisor/supervisor/types/types.go#L258-L266

Reply:
opened issue https://github.com/ethereum-optimism/specs/issues/717